tests/detect-ldap-request-operation/test.rules

```diff
@@ -0,0 +1,9 @@
+alert tcp any any -> any any (msg:"Test LDAP bind request"; ldap.request.operation:0; sid:1;)
```

Comment:
This is good for a first iteration.

In a next iteration, we want to have both signatures ` ldap.request.operation:0;` and ` ldap.request.operation:bind;` (I say `bind` but we should take whatever is being logged as metadata)

See websocket.opcode and usage of `git grep EnumString` in suricata